tests/unit/test_cli_tools.py

```diff
@@ -833,3 +835,195 @@ def test_arg_parser_generate_table_partitions_help(capsys):
+    mock_args.return_value = argparse.Namespace(beta_cmd="new-feature", new_option="value")
+    args = get_parsed_args()
+    assert args.beta_cmd == "new-feature"
+    assert args.new_option == "value"
+
+def test_get_parsed_args_generate_table_partitions(mock_args):
+    """Test arg parser values with generate-table-partitions command."""
+    mock_args.return_value = argparse.Namespace(partition_num=10)
+    args = get_parsed_args()
+    assert args.partition_num == 10
+
+def test_get_parsed_args_query(mock_args):
+    """Test arg parser values with query command."""
+    mock_args.return_value = argparse.Namespace(query="SELECT * FROM my_table")
+    args = get_parsed_args()
+    assert args.query == "SELECT * FROM my_table"
+
+def test_get_parsed_args_add_oracle_connection(mock_args):
+    """Test arg parser values with connections add command for Oracle."""
+    mock_args.return_value = argparse.Namespace(
+        user="my_user",
+        password="my_password",
+        host="my_host",
+        port="1521",
+        database="my_database"
+    )
+    args = get_parsed_args()
+    assert args.user == "my_user"
+    assert args.password == "my_password"
+    assert args.host == "my_host"
+    assert args.port == "1521"
+    assert args.database == "my_database"
+
+def test_get_parsed_args_add_postgresql_connection(mock_args):
+    """Test arg parser values with connections add command for PostgreSQL."""
+    mock_args.return_value = argparse.Namespace(
+        host="localhost",
+        port="5432",
+        user="postgres",
+        password="password"
+    )
+    args = get_parsed_args()
+    assert args.host == "localhost"
+    assert args.port == "5432"
+    assert args.user == "postgres"
+    assert args.password == "password"
+
+def test_get_parsed_args_add_sqlserver_connection(mock_args):
+    """Test arg parser values with connections add command for SQL Server."""
+    mock_args.return_value = argparse.Namespace(
+        host="localhost",
+        port="1433",
+        user="sa",
+        password="password"
+    )
+    args = get_parsed_args()
+    assert args.host == "localhost"
+    assert args.port == "1433"
+    assert args.user == "sa"
+    assert args.password == "password"
+
+def test_get_parsed_args_add_sqlite_connection(mock_args):
+    """Test arg parser values with connections add command for SQLite."""
+    mock_args.return_value = argparse.Namespace(database="my_database.db")
+    args = get_parsed_args()
+    assert args.database == "my_database.db"
+
+def test_get_parsed_args_add_mariadb_connection(mock_args):
+    """Test arg parser values with connections add command for MariaDB."""
+    mock_args.return_value = argparse.Namespace(
+        host="localhost",
+        port="3306",
+        user="root",
+        password="password"
+    )
+    args = get_parsed_args()
+    assert args.host == "localhost"
+    assert args.port == "3306"
+    assert args.user == "root"
+    assert args.password == "password"
```

Comment:
Since DVT doesn't have official support for SQLite and MariaDB, these methods are not needed.

```suggestion
```